jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -53,9 +55,9 @@ class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
     uses_active_status = Bool(True).tag(sync=True)
 
     # feature flag for cone support
-    dev_cone_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
-    dev_bg_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
-    dev_subpixel_support = Bool(False).tag(sync=True)  # when enabling: add entries to docstring
+    dev_cone_support = Bool(True).tag(sync=True)  # when enabling: add entries to docstring
+    dev_bg_support = Bool(True).tag(sync=True)  # when enabling: add entries to docstring
+    dev_subpixel_support = Bool(True).tag(sync=True)  # when enabling: add entries to docstring
```

Comment:
I think this PR fully enables cone and subpixel, but not background, support, right?  If so, please keep `dev_bg_support` defaulting to True, and remove the others entirely (some updates needed to if-statements here and in the vue file).

Reply:
Ok I set it back to False (I assume that is what you meant). The comments say to add entries to the docstring when enabled but should I rename them first? `dev_cone_support` sounds like something in development rather than something that is ready for users.

Reply:
`dev_cone_support` and `dev_subpixel_support` and all mentions of them should be removed entirely.  The `user_api` if-statements can then be combined directly into the `expose` list, and then _those_ items should be added to the API docstring.